README.md

```diff
@@ -20,7 +20,7 @@ Make sure your LinkedIn account is **NOT** protected with 2FA
 #### Usage
 > pip install -r requirements.txt
 
-Edit config.py file (username, password and courses slugs)  
+Copy and edit `config.py.example` (username, password, and courses slugs)

```

Comment:
The entry in .gitignore was added after config.py was checked-in so it still was included in git.